tools/build/build.js

```diff
@@ -226,6 +230,13 @@ switch (BUILD_MODE) {
       taskDm('CBT','CIBUILDING')
     ];
     break;
+  case NO_DM_BUILD:
+    tasksToRun = [
+      taskYarn,
+      taskTgfont,
+      taskTgui
```

Comment:
This sequence is at the start of every build mode. Can we de-duplicate it?

Reply:
Sure